function/pkg/resources/unstructured/git_repository.go

```diff
@@ -0,0 +1,20 @@
+package unstructured
+
+import (
+	"github.com/kyma-incubator/hydroform/function/pkg/workspace"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const gitRepositoryApiVersion = "serverless.kyma-project.io/v1alpha1"
+
+func NewPublicGitRepository(cfg workspace.Cfg) (out unstructured.Unstructured, err error) {
```

Comment:
Why not use https://github.com/kyma-project/kyma/blob/d604878d8999b3b5e1282a8431e55afb1384adb0/components/function-controller/pkg/apis/serverless/v1alpha1/gitrepository_types.go#L8 ?

Reply:
https://github.com/kyma-incubator/hydroform/pull/129#discussion_r498808362